pylint_odoo/test_repo/test_module/except_pass.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+"""Test Except Pass usage"""
+
+
+class TestExceptPass(object):
+    """Test Except Pass class """
+
+    def test_method(self):
+        """Test method """
+        try:
+            raise Exception('Exception')
+        except Exception:
```

Comment:
Add other test case when there aren't handlers.
```python
       try:
            raise Exception('Exception')
       except Exception:
```

and other ones using many handlers

```python
       try:
            raise Exception('Exception')
       except Exception, UserError:
```


```python
       try:
            raise Exception('Exception')
       except (Exception, UserError) as exception: